app/views/maths_and_physics/claims/teaching_maths_or_physics.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for(:page_title, page_title(t("maths_and_physics.questions.teaching_maths_or_physics"), policy: current_policy_routing_name, show_error: current_claim.errors.any?)) %>
+<% path_for_form = current_claim.persisted? ? claim_path(current_policy_routing_name) : claims_path(current_policy_routing_name) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: { "eligibility.teaching_maths_or_physics": "claim_eligibility_attributes_teaching_maths_or_physics_true" }) if current_claim.errors.any? %>
+    <%= form_for current_claim, url: path_for_form  do |form| %>
+      <%= form_group_tag current_claim do %>
+        <%= form.fields_for :eligibility, include_id: false do |fields| %>
+
+          <%= fields.hidden_field :teaching_maths_or_physics %>
+
+          <fieldset class="govuk-fieldset">
+
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+              <h1 class="govuk-fieldset__heading">
+                <%= t("maths_and_physics.questions.teaching_maths_or_physics") %>
+              </h1>
+            </legend>
+
+            <%= errors_tag current_claim.eligibility, :teaching_maths_or_physics %>
+
+            <div class="govuk-radios govuk-radios--inline">
+
+              <div class="govuk-radios__item">
+                <%= fields.radio_button(:teaching_maths_or_physics, true, class: "govuk-radios__input") %>
+                <%= fields.label :teaching_maths_or_physics_true, "Yes", class: "govuk-label govuk-radios__label" %>
+              </div>
+
+              <div class="govuk-radios__item">
+                <%= fields.radio_button(:teaching_maths_or_physics, false, class: "govuk-radios__input") %>
+                <%= fields.label :teaching_maths_or_physics_false, "No", class: "govuk-label govuk-radios__label" %>
+              </div>
+
+            </div>
+
+          </fieldset>
+
+        <% end %>
+      <% end %>
+
+      <%= form.submit "Continue", class: "govuk-button" %>
```

Comment:
I am still convinced we should include `data-module="govuk-button"` on these to properly fire the JS accessibility shim. 

I believe it works by building a list of govuk-buttons on the page from the `data-module` attribute:

https://github.com/alphagov/govuk-frontend/blob/2529efcbdcc66294694d3cd7d86246c0083414f4/src/govuk/all.js#L22

and then checks for `role-button` as well.

https://github.com/alphagov/govuk-frontend/blob/2529efcbdcc66294694d3cd7d86246c0083414f4/src/govuk/components/button/button.js#L21

Reply:
Thanks, will add it!

Reply:
Do we need something in the backlog to update all the rest of the buttons in the same way?

Reply:
Would be good to check, maybe as part of an accessibility sweep – what do you think @titlescreen

Reply:
Yeah that is a good idea, I think it would be worthwhile making something in the backlog and look at this and accessibility when we have an end to end journey. What do you think?